### PR TITLE
[Feature] Update copies for packaging

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

This PR is part of the packaging feature flag.
The changes:
- copy updates
- new alert for a personal account

### Attachments
https://zpl.io/bJLvLv3

### CODE REVIEW

File: Wire-iOS/Resources/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -1651,17 +1651,20 @@
 "feature_config.file_sharing_restrictions.audio" = "Receiving audio files is prohibited";
 
 //Feature flag - Conference calling
-"feature_config.update.conference_calling.alert.title" = "Wire Enterprise";
-"feature_config.update.conference_calling.alert.message" = "Your team was upgraded to Wire Enterprise, which gives you access to features such as conference calls and more.";
-"feature_config.update.conference_calling.alert.message.learn_more" = "Learn more about Wire Enterprise";
+"feature_config.update.conference_calling.alert.title" = "Enterprise plan";
+"feature_config.update.conference_calling.alert.message" = "Your team was upgraded to the Enterprise plan. You now have access to features such as starting conference calls.";
+"feature_config.update.conference_calling.alert.message.learn_more" = "Learn more about Enterprise plan";
```
Comment (johnxnguyen) [inline Wire-iOS/Resources/Base.lproj/Localizable.strings:1656]: I think it should be "the Enterprise plan". (see: https://app.zeplin.io/project/601c1d055a3da08053c6cf71/screen/60f688fc38d3a315c46e264d)

File: Wire-iOS/Sources/UserInterface/MainController/ZClientViewController+ConferenceCallingAvailabilty.swift [hunk 2]
```diff
@@ -76,11 +85,15 @@ extension ZClientViewController: ConferenceCallingUnavailableObserver {
...
+        guard let selfUser = ZMUser.selfUser(),
+              selfUser.hasTeam else {
+            presentConferenceCallingRestrictionAlertForPersonalAccount()
+            return
+        }
+        switch selfUser.teamRole {
+        case .admin, .owner:
             presentConferenceCallingRestrictionAlertForAdmin()
-        } else {
+        default:
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/MainController/ZClientViewController+ConferenceCallingAvailabilty.swift:96]: Maybe you can use just the team role:

```swift
guard let selfUser = ZMUser.selfUser() else { return }

switch selfUser.teamRole {
case .admin, .owner:
  // Present alert for admins
case .member, .partner:
  // Present alert for members
case .none:
  // Present alert for personal users
}
```

File: Wire-iOS/Resources/Base.lproj/Localizable.strings (hunk 1 above)
Comment (KaterinaWire) [reply Wire-iOS/Resources/Base.lproj/Localizable.strings]: Fixed

File: Wire-iOS/Sources/UserInterface/MainController/ZClientViewController+ConferenceCallingAvailabilty.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/MainController/ZClientViewController+ConferenceCallingAvailabilty.swift]: Done

**Review by johnxnguyen — APPROVED** [review]

**Review by billypchan — APPROVED** [review]

**Pull request merged**